dali/operators/reader/tfrecord_reader_op.cc

```diff
@@ -36,24 +47,22 @@ The index files can be obtained from TFRecord files by using the ``tfrecord2idx`
 that is distributed with DALI.)code",
       DALI_STRING_VEC);
 
-DALI_SCHEMA(_TFRecordReader)
+// Internal readers._tfrecord schema.
+DALI_SCHEMA(readers___TFRecord)
   .DocStr(R"code(Reads samples from a TensorFlow TFRecord file.)code")
-  .OutputFn([](const OpSpec &spec) {
-      std::vector<std::string> v = spec.GetRepeatedArgument<std::string>("feature_names");
-      return v.size();
-    })
+  .OutputFn(TFRecordReaderOutputFn)
   .NumInput(0)
   .AddArg("feature_names", "Names of the features in TFRecord.",
       DALI_STRING_VEC)
   .AddArg("features", "List of features.",
       DALI_TF_FEATURE_VEC)
-  .AddParent("_TFRecordReaderBase")
+  .AddParent("readers___TFRecordBase")
   .AddParent("LoaderBase")
   .MakeInternal();
 
-// Schema for the actual TFRecordReader op exposed
-// in Python. It is here for proper docstring generation
-DALI_SCHEMA(TFRecordReader)
+// Schema for the actual readers.tfrecord op expose in Python.
```

Comment:
```suggestion
// Schema for the actual readers.tfrecord op exposed in Python.
```

Reply:
As this is a comment, and the CI appears to be slow, I will fix it in a separate one.